Tests/Realm.Tests/Database/CollectionTests.cs

```diff
@@ -16,10 +16,13 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#nullable enable
```

Comment:
Let's not use nullable reference annotations yet - while I agree we should get it done at some point, I'd prefer if we did it for all our code at the same time rather than bit by bit.